lib/timeline/component/CurrentTime.js

```diff
@@ -111,7 +111,7 @@ class CurrentTime extends Component {
       if (this.options.rtl) {
         this.bar.style.right = `${x}px`;
```

Comment:
Why is this still using `right` and not transform?

Reply:
Good point. i think RTL is ignored in all these changes.